Cargo.toml

```diff
@@ -1,5 +1,5 @@
 [workspace]
-members = ["limitador", "limitador-server"]
+members = ["limitador", "limitador-server", "limitador-server/sandbox/loadtest"]
```

Comment:
I wouldn't add this to the workspace here... There is no dependency and `loadtest` is really not useful in the context of limitador work, yet that'd impose build times et al...